UIKitLesson1/Helper/WishListVC+.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  AlertController.swift
+//  UIKitLesson1
+//
+//  Created by Kirill Dev on 15.01.2023.
+//
+
+import UIKit
+
+extension WishListViewController {
```

Comment:
```suggestion
extension WishListViewController {
```

https://stackoverflow.com/questions/60827313/how-to-create-extension-for-uialertcontroller-in-swift

почитай плиз еще раз и поправь